nmdc_server/app.py

```diff
@@ -35,18 +49,16 @@ def create_app(env: typing.Mapping[str, str]) -> FastAPI:
         version=__version__,
         docs_url="/api/docs",
         openapi_url="/api/openapi.json",
+        debug=True,
```

Comment:
This should probably be a setting I think?

Reply:
Yes, makes sense to me. Fixed in [41f49af](https://github.com/microbiomedata/nmdc-server/pull/1420/commits/41f49af72185d4e41ffec18b525e6dab1a329d9e)